engine/src/test/java/io/camunda/zeebe/engine/util/TestStreams.java

```diff
@@ -13,6 +13,8 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+import io.camunda.zeebe.auth.api.JwtAuthorizationBuilder;
+import io.camunda.zeebe.auth.impl.Authorization;
```

Comment:
❓ I'm a bit confused about [this](https://github.com/camunda/zeebe/pull/14467/commits/fb4c91bc7560783aa148132fedd1fa076824ca7b) commit. It mentions that you made the job stream tests tenant-aware, but all that's changes is a few imports were added.

Reply:
I think this one got included with all the rebasing I did between branches. I'll clean it up.